### PR TITLE
Add Octopus Cloud FAQ

### PR DESCRIPTION
**Pull request by adrianparker**
Initial PR for new Octopus Cloud FAQ page & content.

### CODE REVIEW

File: src/pages/docs/octopus-cloud/frequently-asked-questions.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+Our information security management system has been assessed and found to conform with ISO 27001:2013. You can view our certificate via our [Trust Center](https://octopus.com/company/trust).
+
+### Is Octopus Cloud compliant with HIPAA?
+Octopus Deploy is not a covered entity under HIPAA. We do not process personally identifiable health data directly or on behalf of our customers. Octopus Cloud has global healthcare industry customers operating on our platform. 
+
+### Where can I learn more about Octopus Deploy compliance?
+The security and privacy of our customers' personal, company, and intellectual property data are top priorities at Octopus. We're dedicated to maintaining and continuously improving our security and compliance programs, which we publish on our [Trust Center](https://octopus.com/company/trust).
+
+### Which authentication providers are supported?
+Octopus Cloud supports the most common [authentication providers](/docs/security/authentication/auth-provider-compatibility#login-support) out of the box.
```
Comment (skyemcleman) [inline src/pages/docs/octopus-cloud/frequently-asked-questions.md:102]: It only makes sense to describe 'Octopus Deploy' (i.e. the overall product encapsulating both Octopus Server and Octopus Cloud) as supporting the most common auth providers out of the box, i.e. like this doc: https://octopus.com/docs/security/authentication.

Octopus Cloud doesn't support Active Directory or LDAP Authentication as shown in the table this section links to.

Comment (adrianparker) [reply src/pages/docs/octopus-cloud/frequently-asked-questions.md:102]: Will tighten the answer.

**Review by skyemcleman — APPROVED** [review]
LGTM, just a couple of minor suggestions

File: src/pages/docs/octopus-cloud/frequently-asked-questions.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+### What level of encryption is used between an Octopus Server and a Tentacle?
+All communication between the Tentacle and Octopus is performed over a secure TLS connection. We recommend TLS1.2+. We configure the [trust relationship](/docs/security/octopus-tentacle-communication) between Octopus Server and a Tentacle using public-key cryptography. The server and Tentacle establish the trust relationship using [X.509 self-signed certificates](/docs/security/octopus-tentacle-communication#Octopus-Tentaclecommunication-Octopuscertificates) generated upon installation and using 2048-bit private keys.
+
+### How secure is Tentacle communication?
+Octopus Cloud communicates with Tentacles you deploy on your targets via SSL-encrypted data transmitted over a TCP connection. You can set up [Tentacle communication](/docs/infrastructure/deployment-targets/tentacle/tentacle-communication) modes as listening (acting as the TCP server) or polling (acting as the TCP client). Many customers prefer polling for greater security of the Tentacle host.
```
[the remark this answers is not in the document]
Comment (adrianparker) [reply src/pages/docs/octopus-cloud/frequently-asked-questions.md]: Updated, thank you

**Pull request merged**